crowbar_framework/lib/barclamp_catalog.rb

```diff
@@ -64,12 +64,14 @@ def category(barclamp)
   end
 
   def display_name(barclamp)
-    display = barclamps[barclamp]['display']
-
-    if display.nil? or display.empty?
-      barclamp.titlecase
+    if barclamps[barclamp]
+      if barclamps[barclamp]['display'].blank?
+        barclamp.titlecase
+      else
+        barclamps[barclamp]['display']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.